integration/kubernetes/singleMaster-hdfsJournal/config.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+#
+# This script generates a Docker image for the repository from its current state.
```

Comment:
copy paste mistake?

Reply:
same here

Reply:
Clearly my eyes have problems... I kept thinking you are point to `alluxio-configMap.yaml`